docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
-processor to be working on potentially different instructions at the same time.
+import RPC from "./../../components/RPC-Connection";
 
-Bundles of state transitions represented as blocks may be processed similarly. In the context of
-Polkadot, it aims to increase the throughput of the entire network.
+:::info Learn about Parachain Consensus
+
+To fully follow the material on this page, it is recommended to be familiar with the primary stages
+of the [Parachain Protocol](./learn-parachains-protocol.md).
+
+:::
+
+## Synchronous Backing
+
+Before diving into asynchronous backing, it is important to understand what synchronous backing is
+and what its main limitations are.
+
+:::info What is backing?
+
+**Backing** refers to the process in which a parablock is verified by a subset of validators who
+determine the block’s validity. It is an important step in the validation process for parablocks, as
+it is the first line of defense in ensuring valid state transitions. Validators who back the
+parablock put their stake at risk, and if the block is later found invalid, those validators are
+slashed.
+
+:::
+
+In synchronous backing, parablock generation is tightly coupled to the relay chain's progression on
+a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
+(six-second window), and (if successfully backed) it will be included in a relay-chain block (often
+referred to as the **parent** block, as the parablock anchors itself to it) after an additional six
+seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
+after including the previous one.
+
+The **contextual execution** of new parablocks are parablock ancestors included in the relay chain
+after 12 seconds, where contextual execution refers to the context built by included parablock
+ancestors used by the parachains to generate new parablocks.
+
+:::info Candidate Recepts
+
+- Not the entire parablock, but the **candidate receipt** consisting of the hash of the parablock,
+  state roots and ID info is placed on the parent block on the relay chain.
+- The relay chain does not access the entire state of a parachain but only the values that changed
+  during that block and the merkelized hashes of the unchanged values.
+
+:::
+
+### Synchronous Backing Mechanics
 
-:::info
+In synchronous backing we can imagine parablocks on a conveyor belt with the following properties:
 
-In order to realize which exact processes this upgrade improves, it is recommended to be familiar
-with the stages of parablock validation, which you may [read here](./learn-parachains-protocol.md).
+- the belt is synched with the relay chain: every parablock is generated and backed every 6 seconds,
+  and included every 12 seconds
+- the belt can carry only one parablock at a time (no [pipelining](#pipelining))
+
+The diagram below shows parablocks on their way from being generated to being backed and included
```

Comment:
I'd suggest stripping out the visualization of how full of transactions parablocks are to begin with. Also you could separate out the basic pipelining visual on the right into its own graphic to be shown first. The end result would be three different visuals with increasing complexity:
1. Simple pipelining table
2. This graphic without parablock transaction fill accounted for
3. This graphic in full